src/Copy/BrandCopy.elm

```diff
@@ -620,4 +666,4 @@ relatedInfo storyId =
             [ 1, 4 ]
 
         _ ->
```

Comment:
You'll probably want to add a `4` case - so it's explicit. The `_` is a catchall.

Reply:
yes, I thought so but every time I change, I get an error:

![Peek 2020-01-14 16-26](https://user-images.githubusercontent.com/29477363/72362287-cd219a00-36ea-11ea-9aba-a45c89744682.gif)

Reply:
You can't remove the `_`

Reply:
Elm requires every possible path to be covered. Since we allow any Int in that path, we have to cover the case of anything else.

Reply:
ahh, ok, noted.